Source/Editor/Options/InterfaceOptions.cs

```diff
@@ -340,6 +340,13 @@ public FontReference OutputLogTextFont
         [EditorDisplay("Output Log", "Scroll to bottom"), EditorOrder(470), Tooltip("Scroll the output log view to bottom automatically after new lines are added.")]
         public bool OutputLogScrollToBottom { get; set; } = true;
 
+        /// <summary>
+        /// Gets or set a value indicating wether the outpug log panel should be focussed on begin play.
+        /// </summary>
+        [DefaultValue(false)]
+        [EditorDisplay("Output Log", "Focus Output Log Panel on Begin Play"), EditorOrder(471), Tooltip("Focus the Output Log panel on Begin Play. Focuses the previously selected panel on End Play.")]
+        public bool FocusOutputLogOnBeginPlay { get; set; } = false;
```

Comment:
I think it would be better to use `Show` instead of `Focus`  as there already is `FocusOnPlayMode` option enum.